### DISCUSSION TITLE
Static prompt location?

### DISCUSSION THREAD

**Discussion by daytonjones** [python-cmd2/cmd2#1132] [Q&A]
Is there a method, or easy way, to always have the prompt located at the bottom of the screen/terminal?

**Comment by tleonhardt** [comment]
No, by nature of `cmd2` extending `cmd`, it is a line-oriented command interpreter.

However, there are various options you might want to look into.  See the [async_printing.py](https://github.com/python-cmd2/cmd2/blob/master/examples/async_printing.py) example.
